src/client/_clientwindow.py

```diff
@@ -1,66 +1,49 @@
-from functools import partial
-
-from PyQt5.QtCore import QUrl, QProcess
-from PyQt5.QtWidgets import QLabel, QMessageBox
+from PyQt5 import QtCore, QtWidgets, QtGui
 
-import config
+from config import Settings, is_beta, VERSION
```

Comment:
I'd rather we kept this import as it was, "is_beta" and "VERSION" can be easily confused with local variables.

Reply:
ok